queue.go

```diff
@@ -228,7 +221,7 @@ func (api *API) GetQueue(ctx context.Context, rc *ResourceContainer, queueName s
 // UpdateQueue updates a queue.
 //
 // API reference: https://api.cloudflare.com/#queue-update-queue
-func (api *API) UpdateQueue(ctx context.Context, rc *ResourceContainer, params UpdateQueueParams) (Queue, error) {
+func (api *API) UpdateQueue(ctx context.Context, rc *ResourceContainer, queueName string, params UpdateQueueParams) (Queue, error) {
```

Comment:
the three parameter method signatures were intentional for the next stage of the SDKs. you can read more about it at https://github.com/cloudflare/cloudflare-go/blob/master/docs/experimental.md

Reply:
I see. So how should I pass both parameters (existing queue name, and new queue name) into the function?
I could change `UpdateQueueParams` to something like:
```
type UpdateQueueParams struct {
        ExistingName string `json:"-"`
	UpdatedName string `json:"queue_name,omitempty"`
}
```

Then to make the API call:
```
	uri := fmt.Sprintf("/accounts/%s/workers/queues/%s", rc.Identifier, params.ExistingName)
	res, err := api.makeRequestContext(ctx, http.MethodPut, uri, params)
```

Would that be okay?

Reply:
yep, that seems fine. I would go with `Name` and `UpdatedName` personally but either way. 

we generally avoid this sort of scenario because we don't make the URL resource identifier modifiable. it remains the same for the life of the resource. if we want to find by an attribute like name, it is a wiry parameter on a list endpoint.

Reply:
👍 Thanks, switched over to use the three parameter pattern.